src/core/expression.js

```diff
@@ -140,6 +140,9 @@ const expressionHelper = {
             } else {
                 resultString = fullString;
             }
+            if (resultString.includes('e+')) {
+                resultString = resultString.replace('e+', '*10^');
+            }
```

Comment:
I implemented it as suggested, but with this solution, we ended up with the power of the number having a "+" operator at the beginning, for example: 555^+10. Therefore, in the `extractExponent` function, I added the condition `term.type !== types.operator` to avoid pushing the operator element into the power.
<img width="124" alt="Zrzut ekranu 2024-05-6 o 15 24 46" src="https://github.com/oat-sa/tao-calculator-fe/assets/133767102/a2f17e4f-20fa-40a1-b85e-e9f711d90c6a">

Reply:
Is the plus in front of the value a real problem? It is valid mathematically. Unfortunately, the proposed solution is preventing from having sub-expression as exponent...